### PR TITLE
fix: fix docker gw0 mms size for xwfm

### PR DESCRIPTION
**Pull request by aharonnovo**
[xwfm][ci[ - fix docker gw0 mms size

## Test Plan

ci

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by themarwhal — APPROVED** [review]

**Comment by themarwhal** [comment]
Hey @aharonnovo please fix the DCO check failure.

(See here for instructions)
https://github.com/magma/magma/pull/8263#issuecomment-886637789

File: xwf/gateway/integ_tests/gw/entrypoint.sh [hunk 1]
```diff
@@ -63,5 +63,9 @@ dnsmasq
 echo "run DHCP server"
 /usr/sbin/dhcpd -f -cf /etc/dhcp/dhcpd.conf -user dhcpd -group dhcpd --no-pid gw0 &
 
+echo "set mms"
+iptables -t mangle -A FORWARD -o gw0 -p tcp --tcp-flags SYN,RST SYN -j TCPMSS --set-mss 1400
```
Comment (quentinDERORY) [inline xwf/gateway/integ_tests/gw/entrypoint.sh:67]: Can you put long options names too?

File: xwf/gateway/integ_tests/gw/entrypoint.sh [hunk 2]
```diff
@@ -63,5 +63,9 @@ dnsmasq
 echo "run DHCP server"
 /usr/sbin/dhcpd -f -cf /etc/dhcp/dhcpd.conf -user dhcpd -group dhcpd --no-pid gw0 &
 
+echo "set mms"
```
Comment (quentinDERORY) [inline xwf/gateway/integ_tests/gw/entrypoint.sh:66]: Don't you mean mss?

**Review by quentinDERORY — APPROVED** [review]

File: xwf/gateway/integ_tests/gw/entrypoint.sh (hunk 1 above)
Comment (aharonnovo) [reply xwf/gateway/integ_tests/gw/entrypoint.sh:67]: I don't understand

**Comment by aharonnovo** [comment]
abondon since I had mess with verification of the commit, replaced by https://github.com/magma/magma/pull/8282

File: xwf/gateway/integ_tests/gw/entrypoint.sh (hunk 1 above)
Comment (quentinDERORY) [reply xwf/gateway/integ_tests/gw/entrypoint.sh:67]: As a general rule of thumb, use long flags everywhere as it increases readability https://magma.github.io/magma/docs/next/contributing/contribute_conventions#all-documentation

**Pull request closed without merging**